README.md

```diff
@@ -6,39 +6,41 @@
 
 Extends [`stylelint-config-recommended`](https://github.com/stylelint/stylelint-config-recommended).
 
-Turns on additional rules to enforce the common stylistic conventions found within a handful of CSS styleguides, including: [The Idiomatic CSS Principles](https://github.com/necolas/idiomatic-css),
+Turns on additional rules to enforce common conventions found in the specifications and in a handful of CSS styleguides, including: [The Idiomatic CSS Principles](https://github.com/necolas/idiomatic-css),
 [Google's CSS Style Guide](https://google.github.io/styleguide/htmlcssguide.html#CSS_Formatting_Rules), [Airbnb's Styleguide](https://github.com/airbnb/css#css), and [@mdo's Code Guide](https://codeguide.co/#css).
 
-It favours flexibility over strictness for things like multi-line lists and single-line rulesets, and tries to avoid potentially divisive rules.
-
-Use it as is or as a foundation for your own config.
```

Comment:
We can take this line out as we encourage users (lower down the README) to extend the config with more rules that [limit language features](https://stylelint.io/user-guide/rules/list#limit-language-features), e.g. `unit-allowed-list`. These are some of the most useful, unique and powerful rules in stylelint. We should encourage users to investigate them. They tend to be specific to teams and projects, so we can't add them to this config.